recipe/meta.yaml

```diff
@@ -1,38 +1,56 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
+{% set version_str = "v3.0.0-beta-2" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: 42aa7ab79ebcb94917e938ff482ef7a6
+    fn: protobuf_{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
+    md5: e7f2602baffcbc27fb607de659cfbab6
 
 build:
-    number: 2
+    number: 3
+    skip: true  # [win]
 
 requirements:
     build:
+        - autoconf
+        - automake
+        - libtool
+        - pkg-config
         - python
-        - setuptools
+        - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
+        - gcc                # [osx]
```

Comment:
I'm missing something. the `-stdlib=libstdc++` is used to tell clang to use the libstdc++ (which still has old TR1 stuff) for it's standard library rather than the newer, C++11 only libc++. I'm confused what that has to do with Apple's gcc.